src/controller/data_model/controller-clusters.matter

```diff
@@ -9134,6 +9134,56 @@ provisional cluster ContentAppObserver = 1296 {
   command ContentAppMessage(ContentAppMessageRequest): ContentAppMessageResponse = 0;
 }
 
+/** Supports the ability for clients to request the commissioning of themselves or other nodes onto a fabric which the cluster server can commission onto. */
+cluster CommissionerControl = 1873 {
+  revision 1; // NOTE: Default/not specifically set
```

Comment:
please explicitly set the revision in the XML file (even if this is revision 1) to ensure this message goes away and we explicitly start tracking it.